src/include/module/module/interface.h

```diff
@@ -97,7 +107,7 @@ struct module_interface {
 	 * at least IBS bytes of data on first source and at least OBS free space on first sink
 	 *
 	 * In case more sophisticated check is needed the method should be implemented in
-	 * the module
+	 * the module. Usually can be __cold
```

Comment:
Isn't this a hot-path call (so should not be cold)?

Reply:
@kv2019i right, I suppose it is, thanks for catching!